webextension/background.js

```diff
@@ -1,3 +1,63 @@
+browser.contextMenus.create({
+  id: "open-in-this-container",
+  title: "Always Open in This Container",
+  contexts: ["all"],
+});
+
+function getUserContextIdFromCookieStore(cookieStore) {
+  const container = cookieStore.replace("firefox-container-", "");
+  if (container !== cookieStore) {
+    return container;
+  }
+  return false;
+}
+
+const storagePrefix = "originMap@@_";
+const storageArea = browser.storage.local;
+
+browser.contextMenus.onClicked.addListener((info, tab) => {
```

Comment:
Do we need to add:
```
if (info.menuItemId !== "open-in-this-container") {
  return;
}
```
To make sure that clicks on other context menus don't trigger the assignment?